src/util/formattingNodeToHtml.ts

```diff
@@ -3,17 +3,38 @@ import _ from 'lodash'
 import stripStyleAttribute from './stripStyleAttribute'
 
 /** Strip span and encode a <i> or <b> element as HTML. */
+const replaceFontToSpan = (node: Element) => {
+  if (node.tagName === 'font') {
```

Comment:
Mutating arguments is considered a bad practice. Please find a way to do this in a more functional style.